tests/ui/trait-bounds/enum-unit-variant-trait-bound.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
```

Comment:
```suggestion
// Regression test for one part of issue #105306.

fn main() {
```

Idk, your call

Reply:
hehe i just copied it from the other pr, but yeah, i'll add it!